src/naev.c

```diff
@@ -422,11 +422,13 @@ int main( int argc, char** argv )
    /* Tell the player to migrate their configuration files */
    /* TODO get rid of this cruft ASAP. */
```

Comment:
Probably safe to get rid of this no? It was introduced for a breaking change, but it's been like this forever, so we might as well remove it.

Reply:
Good riddance.